winpe/README.md

```diff
@@ -0,0 +1,81 @@
+#Editing the WINPE Operating System for windows installation
+
+
+###Download Windows ADK (Assessment and Deployment Kit)
+It is recommended that you use Windows ADK for windows 10
+https://developer.microsoft.com/en-us/windows/hardware/windows-assessment-deployment-kit 

```

Comment:
The powershell commands below reference version 8.1.  Is that the right version number for the windows 10 ADK that you reference here?